.rubocop_fixme.yml

```diff
@@ -23,7 +23,7 @@ Metrics/AbcSize:
 
 Metrics/ModuleLength:
   Exclude:
-    - 'app/controllers/concerns/hyrax/collections_controller_behavior.rb'
+    - 'app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb'
```

Comment:
This is already a discussion I had with @ggeisler who I believe indicated we should us "admin" for any work that had to do with administration of repository objects. This is not an area that is restricted to administrators.